solana-programs/payment-router/programs/payment-router/src/error.rs

```diff
@@ -0,0 +1,15 @@
+use anchor_lang::prelude::*;
+
+#[error_code]
+pub enum PaymentRouterErrorCode {
```

Comment:
Im not too familiar with Solanas #[error_code] macro but itd be nice if we could format some of these error codes so we know which accounts have issues. 

```
#[msg("something {}", _1)]
Variant(String)
```